Examples/test-suite/python/Makefile.in

```diff
@@ -122,6 +123,7 @@ VALGRIND_OPT += --suppressions=pythonswig.supp
 
 # Custom tests - tests with additional commandline options
 python_flatstaticmethod.cpptest: SWIGOPT += -flatstaticmethod
+director_guard.cpptest: SWIGOPT += -threads
```

Comment:
I would prefer your other option you gave, that is, use threads in the `%module` as we only put 'code' into the build system if there is no other reasonable alternative. Everything should really be in in the interface file in an ideal world.

Reply:
I though other languages may use it without threads.
But it does make sense to add it to the interface file.
Other languages as ruby, should implement it in a similar manner with the `threads` option.